net/boinc-wrapper/Makefile

```diff
@@ -0,0 +1,74 @@
+# Copyright 2022 by Vitalii Koshura
```

Comment:
Please use SPDX.

Reply:
What is SPDX?

Reply:
https://spdx.org/licenses/
e.g.

https://github.com/openwrt/packages/blob/1acf423ef5891eb88c8e2bbdf8dad7896b8db398/devel/flex/Makefile#L1-L3

Reply:
Fixed